app/views/shared/_head.html.erb

```diff
@@ -11,7 +11,12 @@
                   odin!
 
 
-Our full stack curriculum is free and supported by a passionate open source community. Do you want to contribute? Chat with us at https://discord.gg/hvqVr6d
+Our full stack curriculum is free and supported by a passionate open source community. Do you want to contribute? Chat with us at https://discord.gg/V75WSQG
+
+codyloyd
```

Comment:
?? What's this

Reply:
Uhh, good question. I'd assume a miss paste from when I was copying the URL to search for it

Reply:
@codyloyd Fixed